client/components/open/forms/OpenForm.vue

```diff
@@ -451,10 +451,47 @@ export default {
     },
     handleDragDropped(data) {
       if(data.added){
-        this.workingFormStore.addBlock(data.added.element, data.added.newIndex)
+        let targetIndex = 0;
+        const newFields = clonedeep(this.workingFormStore.content.properties)
+        const groups = []
+        let currentArray = [];
+        // Group into pages by page break
+        newFields.forEach(item => {
```

Comment:
We already have `fieldGroups` for this on this componenent.. No need to duplicate properties etc
Also, all of this should only run if `currentFieldGroupIndex` > 0